wiki/Contests/Monthly_Beatmapping_Contest/en.md

```diff
@@ -0,0 +1,79 @@
+Submissions for the MBC must abide by the following rules:
+
+- **Submissions must follow the month's beatmapping limitation.** The limitation differs month by month and is detailed on each contest's announcement post.
+- **Submissions must be a single difficulty from one of the month's selected songs.** Selected songs differ month by month and are included in each contest's announcement post.
+- **Submissions must be in `.osz` format.** Storyboarded content will not be considered.
+- **Submissions must be in the osu! game mode.** Other game modes will be considered after trialing this contest with osu! only.
+- **Submissions must *not* be uploaded to the osu! website through the Beatmap Submission System (*BSS*).** Creators of each entry are anonymous during the screening and voting processes.
+- **Submissions must abide by the [ranking criteria](/wiki/Ranking_Criteria).**
+
+One or more rules may be added or dropped depending on the month's specifications. Any exceptions will be clearly explained in the contest's announcement post.
+
+## Results
+
+Results for the MBC are first narrowed down to the most competitive entries, then published for community vote. Creators of all entries are anonymous in the screening and voting stages.
+
+### Screening
+
+A group of at least 10 experienced beatmappers (*screening crew*) are tasked with reviewing all entries and providing their top picks in order of preference. Entries that receive the most praise from the screening crew will move on to the voting stage.
+
+### Voting
+
+The amount of beatmaps that advance to the voting stage varies depending on the amount of highly competitive submissions the contest receives and will usually range between 5 and 10 finalists.
+
+Voting takes place on the respective contest's [Community Contests page](https://osu.ppy.sh/community/contests). Users are encouraged to vote for their favorite entry and to look out for any entries that effectively work with the month's beatmapping limitation.
+
+### Previous contests
+
+Information and results for all iterations of the MBC since its relaunch in 2020 can be found below.
+
+| Contest Listing | Winner | Runner-up | Screening crew's #1 | Results Post |
+| :-- | :-- | :-- | :-- | :-- |
+| [January 2020](https://osu.ppy.sh/community/contests/63) | TBD | TBD | TBD | TBD |
+
+## Rewards
+
+The Monthly Beatmapping Contest rewards users with points for different types of results (*MBC points*):
+
+- **Community vote winner:** 2 points
+- **Community vote runner-up:** 1 point
+- **Screening crew's #1 pick:** 1 point
+
+MBC points reward users at certain thresholds:
+
+- **Profile badge:** badges are attainable at 3, 6, and 10 points each. The badges are tiered, meaning each new badge will replace the previous one (and they get progressively shinier!).
+- ***Elite Mapper* forum title:** attainable at 10 points.
+- **osu!supporter:** each point earned provides a user with a month of supporter for themself or someone of their choosing.
```

Comment:
should be written as normal sentences instead of with a colon, with the current wording. like the first could be
```markdown
- **Profile badges** are attainable at 3, 6, and 10 points each. The badges are tiered, meaning each new badge will replace the previous one (and they get progressively shinier!).
```